app/src/main/java/atlant/moviesapp/fragments/AiringTvFragment.java

```diff
@@ -106,13 +173,23 @@ public void showTvShows(final List<TvShow> data) {
 
     @Override
     public void showProgress() {
-        progressBar.setVisibility(View.VISIBLE);
+        if (progressBar != null)
+            progressBar.setVisibility(View.VISIBLE);
 
     }
 
+    public boolean isNetworkAvailable() {
```

Comment:
A comment has already been written of the usages of the method isNetworkAvailable()